libcrux_crypto/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "openmls_libcrux_crypto"
+version = "0.1.0"
+edition = "2021"
```

Comment:
Could you add some metadata here (authors license, description, etc.) like we have in the other Cargo.tomls?